src/Shells/StrixMusic.Shells.ZuneDesktop/Controls/Views/Collection/ZuneTrackCollectionItem.cs

```diff
@@ -108,16 +102,6 @@ private void DetachEvents(ITrackCollectionViewModel trackCollection)
             trackCollection.TracksChanged -= TrackCollection_TracksChanged;
         }
 
-        private void AttachEvents(ITrack track)
-        {
-            track.ArtistItemsChanged += Track_ArtistItemsChanged;
-        }
-
-        private void DetachEvents(ITrack track)
-        {
-            track.ArtistItemsChanged -= Track_ArtistItemsChanged;
-        }
-
         private void TrackCollection_TracksChanged(object sender, IReadOnlyList<OwlCore.Events.CollectionChangedItem<ITrack>> addedItems, IReadOnlyList<OwlCore.Events.CollectionChangedItem<ITrack>> removedItems)
```

Comment:
We don't need this event handler anymore. We're doing functionally the same thing in `ZuneTrackCollection`, and can assign `ShouldShowArtistList` on each `ZuneTrackCollectionItem` from there. Less overhead that way.